js/.eslintrc.js

```diff
@@ -1,69 +1,60 @@
 module.exports = {
-    "env": {
-        "browser": true,
-        "es6": true,
-        "node": true,
-        "worker": true
+    env: {
+        browser: true,
+        es6: true,
+        node: true,
+        worker: true,
     },
-    "extends": "eslint:recommended",
-    "parserOptions": {
-        "ecmaVersion": 8
+    extends: "eslint:recommended",
+    parserOptions: {
+        ecmaVersion: 8,
     },
-    "rules": {
+    rules: {
         "accessor-pairs": "error",
         "array-bracket-newline": "off",
-        "array-bracket-spacing": [
-            "error",
-            "never"
-        ],
+        "array-bracket-spacing": ["error", "never"],
         "array-callback-return": "off",
         "array-element-newline": "off",
         "arrow-body-style": "error",
-        "arrow-parens": [
-            "error",
-            "as-needed"
-        ],
+        "arrow-parens": ["error", "as-needed"],
         "arrow-spacing": [
             "error",
             {
-                "after": true,
-                "before": true
-            }
+                after: true,
+                before: true,
+            },
         ],
         "block-scoped-var": "error",
         "block-spacing": "error",
         "brace-style": "off",
         "callback-return": "off",
-        "camelcase": "off",
+        camelcase: "off",
```

Comment:
I think it's more common to have JSON always require quotes even when not strictly necessary. As such, I wouldn't be opposed to adding this as rule too! Thoughts @externl?

Edit: Just for clarity I am referring to that words like `camelcase` now no longer have double quotes.

Reply:
I agree, do you know the rule for this?

Reply:
No, I think we must get it from somewhere else on the docs. This is its .eslintrc so it could be coming from the `next` config. 

```
{
  "env": {
    "browser": true,
    "es2021": true
  },
  "extends": [
    "next",
    "next/core-web-vitals",
    "plugin:jsx-a11y/strict",
    "prettier",
    "plugin:tailwindcss/recommended"
  ],
  "plugins": [
    "prettier"
  ]
}
```